app/controllers/course/assessment/submissions_controller.rb

```diff
@@ -2,17 +2,31 @@
 class Course::Assessment::SubmissionsController < Course::ComponentController
   before_action :load_submissions
   before_action :add_submissions_breadcrumb
-  before_action :load_group_managers, only: [:pending]
+  before_action :load_group_managers, only: [:pending, :index]
 
   def index # :nodoc:
-    @submissions = @submissions.from_category(category).confirmed
-    @submissions = @submissions.filter_by_params(filter_params) unless filter_params.blank?
-    load_assessments
+    respond_to do |format|
+      format.html
+      format.json do
+        @submissions = @submissions.from_category(category).confirmed
+        @submissions = @submissions.filter_by_params(filter_params) unless filter_params.blank?
+        @submission_count = @submissions.count
+        @submissions = @submissions.filter_by_params(filter_page_num) unless filter_page_num.blank?
+        load_assessments
+      end
+    end
   end
 
   def pending
-    @submissions = pending_submissions.from_course(current_course)
-    load_assessments
+    respond_to do |format|
+      format.html
```

Comment:
This has the same effect as just deleting `pending.html.slim` so I'll just do that instead, since users should not be able to access `courses/:courseId/assessments/submissions/pending` anyway